modules/custom/falcon_commerce/falcon_commerce.install

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for Falcon Commerce.
+ */
+
+use Drupal\commerce_store\Entity\Store;
+use Drupal\commerce_price\Price;
+use Drupal\commerce_product\Entity\ProductVariation;
+use Drupal\commerce_product\Entity\Product;
+
+/**
+ * Implements hook_install().
+ */
+function falcon_commerce_install() {
+  drupal_flush_all_caches();
+
+  // Create default Commerce Store.
+  // The store's address.
+  $address = [
+    'country_code' => 'US',
+    'address_line1' => '8 The Green',
+    'address_line2' => 'Ste A',
+    'locality' => 'Dover',
+    'administrative_area' => 'DE',
+    'postal_code' => '19901',
+  ];
+
+  // The currency code.
+  $currency = 'EUR';
```

Comment:
I'm guessing that the default currency should be configurable during Falcon installation. Do you have any ideas how we can achieve it?

Reply:
I have removed this hook_install as it is not working correctly with `drush config-import`.